source/math/FloatVectorList.ooc

```diff
@@ -381,4 +381,80 @@ FloatVectorList: class extends VectorList<Float> {
 			result add(this[i] clamp(floor, ceiling))
 		result
 	}
+	findMaxIndex: func -> (Int, Float) {
```

Comment:
Sure, but `findMaxIndex()` claims it is just returning an index.
It seems just as easy to do
```ooc
maxIndex := myList findMaxIndex()
maxValue := myList[index]
```